packages/create-itp-react-app/templates/app/src/common/components/atoms/Button.css

```diff
@@ -0,0 +1,103 @@
+.ant-btn {
```

Comment:
cannot imagine that all of this css is needed for that one button? I would keep this to the bare minimum.